tools/serverpod_cli/lib/src/analyzer/entities/entity_analyzer.dart

```diff
@@ -137,6 +137,8 @@ class SerializableEntityAnalyzer {
     }
     var docsExtractor = YamlDocumentationExtractor(yaml);
 
+    _validateEntityProperties(documentContents);
```

Comment:
"properties" sounds a bit like the fields. Maybe `_validateEnitityType` would be better?